rfcs/ambassador-program/index.md

```diff
@@ -0,0 +1,83 @@
+## GraphQL Ambassador Program
+
+The following is the beginnings of a document outlining the goals, outcomes and tasks of an Ambassador program. 
+
+### Alternative Names
+
+* Champion - already used on the technical side - Champions of spec edits
+* Evangelist - usually out of favour nowadays
+* Advocate - suggests a salaried role?
+* Leader, Star, Hero, Captain, Expert - “Star” we have already used in a recognition role
+* GraphQL Fellowship 
+
+> [!NOTE]
+> “Ambassador” seems to be readily accepted, I don't think we need to discuss the term any further.   
+
+### Ambassador Initiative Goals
+
+The overarching goal is to increase GraphQL adoption. The Ambassadors program aims to accomplish this through:
+
+* Brand advocacy - developers organically vouch for and promote GraphQL
+* Social proof - named individuals with high visibility (and a proportion from known corporate brands) as Ambassadors
+* Increased GraphQL content online and off
+* Community building - increase the number of “known faces” within GraphQL, increased approachability (addressing the concern of “lowering the barrier to join an inclusive GraphQL community” raised by the Community WG)  
+
+How can these goals be measured? 
+
+* Engagement metrics - social, chat channels, search, downloads, page views…
+* New community members (across social media comments, Discord users, local gathering attendees, working group members…) 
+* Volume of talks at conferences
+* Volume of articles written
+
+### Ambassador Tasks
+
+As a GraphQL Ambassador, you will be an important touchstone for people interested in GraphQL to learn more about it in a friendly and constructive way. 
+
+One or more of the following:
+
+* Conference & meetup talks on GraphQL
+* Local gathering organization
+* Written GraphQL content
+* Video / audio GraphQL content
+
+### Foundation Tasks
+
+* Grant available for travel expenses for approved conference & meetup talks
+* Microgrant (or honorarium) available for content creation 
+    * Where is this content hosted? 
```

Comment:
I would encourage personnal websites by default. This is easy to do, promotes the personnal branding of the Ambassadors and removes friction. Some of the content could be hosted on graphql.org on a case by case basis but I wouldn't do it the default.

Reply:
Given that Ambassadors are meant to help promote the project and community, and that anyone can submit a PR to graphql.org for publishing why would we want their efforts to drive traffic to a place other than graphql.org?

Reply:
Network effect + diversity + proximity. The tone of voice can be different in a personnal blog. For an example, we can allow content in other languages. But not only, it may be more familiar, meme-y or enthusiastic, I think that's OK and I would encourage the ambassadors to create genuine connections with their audience.

We could encorage graphql.org content throught the existing grant program and use the Ambassador program to reach out to other places.

Reply:
I didn't quite understand why you were pushing personal websites, but now you've made a strong case! Especially with regards to tone, this is a good thing I agree. This content could also then be hosted on somewhere like dev.to where the social sharing is encouraged.

Reply:
@martinbonnin You are assuming that everyone has a personal blog?  This might be a point where we can be less prescriptive and say it's the ambassador's choice whether to publish on the website or their own blog.

Reply:
@doc-jones It doesn't have to be their own domain, CSS and everything, medium, dev.to and/or hashnode would work too. My main point is I'd like content in many languages and many communities. 

Taking a very personal example, my audience is mostly Android developers that have a very low chance of opening a graphql.org link. But could still open a GraphQL post that speaks to them (I least I hope so!). If an amassador has a very strong Python following base, I'd love to plant some GraphQL seeds there, etc... 

Some of the content could be specifically made for graphql.org but I'd rather have be the exception than the default.

Reply:
Why could this not be the ambassador's choice?

Reply:
Of course, could be the ambassador choice. Just if all ambassadors chose graphql.org, it defeats the idea of reaching out to communities outside the core GraphQL community (the communities that wouldn't necessarily follow the GraphQL properties). 

Plus it requires moderation. If we want to scale, we're bound to reach outside of graphql.org. Talking ballpark numbers, if we have 10 ambassadors and everyone posts once a month, that's ~100 posts, which is more that graphql.org can handle IMO. Both in terms of moderation and/or audience attention.